bc_obps/bc_obps/api.py

```diff
@@ -8,7 +8,6 @@
 # print errors for 422 status codes for faster debugging
 @api.exception_handler(ValidationError)
 def custom_validation_errors(request, exc):
-    print(exc.errors)
     return api.create_response(request, {"detail": exc.errors}, status=422)
```

Comment:
I found it really helpful to have the 422 errors printed, otherwise when debugging you don't know which field is giving you trouble. If it's a security thing could we only print when we're in a development environment?

Reply:
I thought it was just a leftover, Sorry! I'm going to add a comment to keep that.